### PR TITLE
✨ Start placement translator

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR is a work-in-progress and shows the begging of the work on implementing the placement translator.

This is built on #229 and I will rebase after #229 merges.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
/cc @waltforme 
@clubanderson

**Comment by MikeSpreitzer** [comment]
The force-push to 3b414c7 is a rebase onto the `main` branch.

File: pkg/placement/what-resolver.go [hunk 1]
```diff
@@ -0,0 +1,715 @@
...
+		wr.notifyReceivers(cluster, epName)
+		return true
+	}
+	// Now we know that ep != nil
+	prevEp := wsDetails.placements[epName]
+	wsDetails.placements[epName] = ep
+	if prevEp == nil {
+		logger.V(3).Info("Starting watching EdgePlacement")
+	} else {
+		whatPredicateChanged := (apiequality.Semantic.DeepEqual(prevEp.Spec.NamespaceSelector, ep.Spec.NamespaceSelector) ||
```
Comment (waltforme) [inline pkg/placement/what-resolver.go:519]: Should it be '_not_ equal' i.e. `!apiequality.Semantic.DeepEqual(...)`?

**Comment by MikeSpreitzer** [comment]
The force-push to 5f8514d fixes the typo noticed by @waltforme above.

File: pkg/placement/what-resolver.go [hunk 2]
```diff
@@ -0,0 +1,715 @@
...
+			apiequality.Semantic.DeepEqual(prevEp.Spec.NonNamespacedObjects, ep.Spec.NonNamespacedObjects))
```
[the remark this answers is not in the document]
Comment (MikeSpreitzer) [reply pkg/placement/what-resolver.go:520]: Fixed in latest push.

**Pull request merged**